docs/fud/circt.md

```diff
@@ -0,0 +1,56 @@
+# CIRCT
+
+An ongoing effort is under way to establish Calyx as a Dialect in the LLVM umbrella project [CIRCT][]. 
```

Comment:
```suggestion
An ongoing effort is under way to establish Calyx as a *dialect* in the LLVM umbrella project [CIRCT][]. 
```

I'd probably not capitalize "Dialect" throughout—I don't think people tend to use it as a proper noun.